components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -214,6 +257,24 @@ export function CreateWorkspacePage() {
                         {isLoading ? "Loading ..." : isStarting ? "Creating Workspace ..." : "New Workspace"}
                     </Button>
                 </div>
+                {existingWorkspaces.length > 0 && (
+                    <div className="w-full flex flex-col justify-end px-6">
+                        <p className="mt-6 text-center text-base">Running workspaces on this revision</p>
+                        <>
+                            {existingWorkspaces.map((w) => {
+                                return (
+                                    <a
+                                        key={w.workspace.id}
+                                        href={w.latestInstance?.ideUrl || `/start/${w.workspace.id}}`}
+                                        className="rounded-xl group hover:bg-gray-100 dark:hover:bg-gray-800 flex"
+                                    >
+                                        <WorkspaceEntry info={w} shortVersion={true} />
+                                    </a>
```

Comment:
**suggestion**: If we follow the dropdown approach, here's how this could look like:

| :a: | :b: |
|-|-|
| <img width="1440" alt="Welcome-1" src="https://user-images.githubusercontent.com/120486/230644187-edd16c3b-e680-4bb8-8fd0-dc140eec0d86.png"> | <img width="1440" alt="Welcome" src="https://user-images.githubusercontent.com/120486/230644183-40b5b46c-fce4-40db-a821-0fdfc4d1f467.png"> |